src/templates/includes/declaration-html.njk

```diff
@@ -28,7 +28,7 @@
           {{ documentReference(decision) }}
           <td class="govuk-table__cell">{{ decisionMatch(decision.match) }}</td>
           <td class="govuk-table__cell">
-            {{ decision.departmentCode }}
+            {{ decision.departmentCode }}{{ decision.departmentCodeDetail }}
```

Comment:
As per the above comments this change can be undone

Reply:
This can be changed to refer to `decision.authority.text`
and line 23 can be changed to:
`data-authority="{{ decision.authority.value }}"`